docs/app-dev/app-architecture.md

```diff
@@ -52,4 +52,4 @@ See the following for more extensive documentation:
 - [Interchain Standard for the Light-Client REST API](https://github.com/cosmos/cosmos-sdk/pull/1617) (legacy/deprecated)
 - [CometBFT RPC Docs](https://docs.cometbft.com/main/rpc/)
 - [CometBFT in Production](../core/running-in-production.md)
-- [ABCI spec](https://github.com/cometbft/cometbft/tree/main/spec/abci)
+- [ABCI spec](../spec/abci/README.md)
```

Comment:
Will this link work from Github? I don't think so.

Reply:
it will not work, good catch. We can also link to the documentation site `https://docs.cometbft.com/main/spec/abci`, what do you think ? I can't think now a way to make it work both ways.